media-gfx/blender/blender-2.79.recipe

```diff
@@ -10,15 +10,18 @@ benefit from its unified pipeline and responsive development process."
 HOMEPAGE="http://www.blender.org/"
 COPYRIGHT="2002-2017 Blender Foundation"
 LICENSE="GNU GPL v2"
-REVISION="1"
+REVISION="2"
 SOURCE_URI="http://download.blender.org/source/blender-$portVersion.tar.gz"
 CHECKSUM_SHA256="a9de03e769a2a4a0bf92186556896c4f4d32fd9ac4480915ae92d7f95b25c899"
 PATCHES="blender-$portVersion.patchset"
 ADDITIONAL_FILES="blender.rdef.in"
 
-ARCHITECTURES="!x86_gcc2 !x86 x86_64"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
 
 PYTHON_VERSION="3.6"
+PY_VENDOR_DIR=`finddir B_SYSTEM_LIB_DIRECTORY`/python${PYTHON_VERSION}/vendor-packages
+NUMPY_FOLDER=`find $PY_VENDOR_DIR -maxdepth 1 -type d |grep numpy`
```

Comment:
Please move these in BUILD() where they are needed. Here they would be evaluated in the context of haikuporter.